application-workloads/sql/sql-reporting-services-sql-server/azuredeploy.json

```diff
@@ -86,21 +86,24 @@
         "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated. Use the defaultValue if the staging location is not secured."
       },
       "defaultValue": ""
-    }    
+    },
+    "storageAccountEndPoint":{
```

Comment:
remove this parameter and use the environment() function to retrieve the endpoint

Reply:
updated with environment() function